server/apps/immich/src/app.module.ts

```diff
@@ -5,7 +5,7 @@ import { AssetModule } from './api-v1/asset/asset.module';
 import { AuthModule } from './api-v1/auth/auth.module';
 import { ImmichJwtModule } from './modules/immich-jwt/immich-jwt.module';
 import { DeviceInfoModule } from './api-v1/device-info/device-info.module';
-import { ConfigModule } from '@nestjs/config';
+import { ConfigModule as NestJSConfigModule } from '@nestjs/config';
```

Comment:
The module name can be change to `ImmichConfigModule` as well to avoid conflict with Nest.js stuff